build.ps1

```diff
@@ -138,6 +140,65 @@ Function BuildProject($path, $message)
     }
 }
 
+Function RunSamples($path, $message)
+{
```

Comment:
```suggestion
Function RunSamples($path, $message) {
```

Reply:
I prefer braces on their own line, even in PowerShell, but if we choose one way to do it, we should be consistent.